pipeline_steps/kibana.groovy

```diff
@@ -15,7 +15,7 @@ def kibana(branch, vm=null){
 
 def kibana_prep(branch){
   dir("kibana-selenium") {
-    git url: "https://github.com/rcbops-qe/kibana-selenium.git", branch: "${branch}"
+    git url: env.KIBANA_SELENIUM_REPO, branch: "${branch}"
```

Comment:
This _was_ blocking https://github.com/rcbops/rpc-gating/pull/203 since https://github.com/rcbops-qe/kibana-selenium didn't have a branch for newton-14.1.  Since we weren't sure how long it'd take to get that branch cut, we thought it'd make sense to make https://github.com/rcbops-qe/kibana-selenium configurable so that we could fork and branch ourselves.

Anyway, this is no longer blocked so there's no urgency in getting this merged.